mvp-spec/README.md

```diff
@@ -37,6 +37,39 @@ Key features of PAF include:
 - signing transmissions and request with private keys
 - making available alongside the ad an audit log of entities involved
 
+## Nodes
+
+### Operator
+
+The operator is responsible for:
+- generating unique user ids
+- storing these ids and their associated preferences
+
+See operator-api.md for details.
```

Comment:
Sorry, but it would useful to preserve the **hyperlinks** to the documents (see previous version of this doc)
Thanks

Reply:
Indeed, updated.